src/nemos/basis/_decaying_exponential.py

```diff
@@ -40,8 +41,8 @@ def __init__(
         mode="eval",
         label: Optional[str] = "OrthExponentialBasis",
     ):
+        AtomicBasisMixin.__init__(self, n_basis_funcs=n_basis_funcs)
```

Comment:
I think we've already discussed this, but there's no way to have `super().__init__()` initialize both super classes?

Reply:
This modification is quite hard and requires some precision, so we can leave it for the future.

The method that in theory can work is the first one `super().__init__()` however, for this to work, for all the classes inheriting directly `AtomicBasis` and `Basis`, we need to invert the inheritance order. 

This also doesn't work directly, because `AtomicBasis` is using some parameters that are defined at basis initialization.

I think this all can be smoothed out but requires precision and a dedicate PR. I'll add it to an issue